src/WebJobs.Script/WebJobs.Script.csproj

```
@@ -47,7 +47,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.20.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11938" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11952" />	  
```

nit: remove whitespace
```suggestion
    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11952" />
```